platform/view/services/comm/host/rest/websocket/stream.go

```diff
@@ -126,7 +126,9 @@
 		logger.Debugf("Wrote to [%s@%s], but message not ready yet.", s.info.RemotePeerID, s.info.RemotePeerAddress)
 		return n, nil
 	}
-	logger.Debugf("Ready to send to [%s@%s]: [%s]", s.info.RemotePeerID, s.info.RemotePeerAddress, logging.Base64(content))
+	if logger.IsEnabledFor(zapcore.DebugLevel) {
```

Comment:
logging.Base64 return a struct whose String function returns the encoding. The String function is not evaluated if the log level is not debug. So, the additional check is redundant  here.